library/detector.py

```diff
@@ -164,10 +164,10 @@ async def scam(message, cached_messages): # pylint: disable=too-many-return-stat
             return True
 
     for url in urls:
-        parsedurl = urlparse(url)
-        parsedurl = parsedurl.path + parsedurl.query
+        parsed_url = urlparse(url)
+        parsed_url = parsed_url.path + parsed_url.query
 
-        if any(ext in parsedurl for ext in ('.exe', '.msi', '.zip', '.rar')):
+        if any(ext in parsed_url for ext in {'.exe', '.msi', '.zip', '.rar'}):
             await reportmessage(report)
```

Comment:
while youre at it, this might be good to edit

Reply:
Hey sorry, for the really delayed response to this.

Whilst not general practice, the style of this code is to use _ in variable names as little as possible.

In terms of changing the tuple to a set, that is consistent with other data structure choices but pylint nagged about it so I just went with a tuple.

```
Use a sequence type when iterating over values
```